src/components/shoppingListGrouping/shoppingListGrouping.module.css

```diff
@@ -0,0 +1,45 @@
+.root {
+  margin: 0 auto;
+}
+
+.noLists {
```

Comment:
I guess the "right" way of doing this would be to render a component with different root styles but it was just a humble `<p>` so I didn't want to create a whole new component for it.